7forecast.tex

```diff
@@ -34,41 +33,41 @@
 		
   1 & \hyperlink{Benchmark}{Benchmarks/Reference Points}:\hypertarget{Bmark_RefPoints}{} & \multirow{1}{1cm}[-0.1cm]{\parbox{12cm}{SS3 checks for consistency of the forecast specification and the benchmark specification. It will turn on benchmarks if necessary and report a warning.}} \Tstrut\\
     & 0 = skip/omit; & \\
-    & 1 = calculate F\textsubscript{SPR}, F\textsubscript{Btarget}, and F\textsubscript{MSY}; & \\
-    & 2 = calculate F\textsubscript{SPR}, F\textsubscript{MSY}, F\textsubscript{0.10}; and & \\
-    & 3 = add F at B\textsubscript{LIMIT} \\ 
+    & 1 = calculate $F_{SPR}$, $F_{Btarget}$, and $F_{MSY}$; & \\
+    & 2 = calculate $_{SPR}$, $F_{MSY}$, $F_{0.10}$; and & \\
+    & 3 = add $F$ at $B_{LIMIT}$ \\ 
```

Comment:
Why is "LIMIT" in all caps?

Reply:
I think having it in caps is somewhat standard when listing reference and or rebuilding metrics.  I would suggest also having target as TARGET as well.